src/js/shariff.js

```diff
@@ -112,6 +112,12 @@ Shariff.prototype = {
 
         flattrUser: null,
 
+        createLinkIcon: function($shareLink, service) {
+            if (typeof service.faName !== 'undefined') {
+                $shareLink.prepend('<span class="fa ' +  service.faName + '">');
```

Comment:
keep in mind that the `<i>` tag may cause problems on some pages when used. a neutral `<span>` is indeed less intrusive